python/reg_interface/common/reg_base_ops.py

```diff
@@ -12,7 +12,10 @@
   wReg = lib.putReg
   wReg.argtypes=[c_uint,c_uint]
 else:
-  lib = CDLL("librwreg.so")
+  try:
```

Comment:
Please insert here a comment that this has to be removed in future after proper refactoring.
As I can recall, the issue is that our gem-specific tools has been moved out of reg_utils to keep it basic, but it happened after the legacy branch off, that's why the discrepancy appeared.

Reply:
Done